src/OpenSage.Game/Logic/Object/Behaviors/ParkingPlaceBehavior.cs

```diff
@@ -245,13 +248,29 @@ internal override void Load(StatePersister reader)
                 throw new InvalidStateException();
             }
 
-            reader.SkipUnknownBytes(29);
+            reader.SkipUnknownBytes(15);
 
-            var unknown3 = 0x3FFFFFFFu;
-            reader.PersistUInt32(ref unknown3);
-            if (unknown3 != 0x3FFFFFFF)
+            reader.PersistListWithByteCount(_healingData, (StatePersister persister, ref HealingData item) =>
             {
-                throw new InvalidStateException();
+                persister.PersistObjectValue(ref item);
+            });
+
+            reader.SkipUnknownBytes(13);
+
+            // This is the default 0x3FFFFFFFu, unless we are healing something,
+            // in which case it's CurrentFrame + a small number of frames.
```

Comment:
Looks like the healing interval (based on the moduledata) is per second - any chance this is the "next heal frame" or something?

Reply:
Good idea, that's quite likely - https://github.com/OpenSAGE/OpenSAGE/pull/953/commits/5948e1de5420a022201e253c259ef094a84bf2ba